docs/contributing.md

```diff
@@ -0,0 +1,136 @@
+### Latest greatest KubeSaw for AppStudio
+Run the following to install the latest greatest KubeSaw operators in dev mode for AppStudio environment:
+```
+make appstudio-dev-deploy-latest
+```
+
+### Local version
+If you want to install a local version of a specific KubeSaw operator in dev mode then:
+
+1. [Configure your quay account for dev deployment](quay-repo-config.md)
+2. Run any from the following commands:
+```bash
+# To deploy local versions of all repositories:
+make dev-deploy-e2e-local
+
+# To deploy local version only of the host-operator repo:
+make dev-deploy-e2e-host-local
+
+# To deploy local version only of the member-operator repo:
+make dev-deploy-e2e-member-local
+
+# To deploy local version only of the registration-service repo:
+make dev-deploy-e2e-registration-local
+```
+
+### ToolchainStatus
+Run the following command:
+```shell
+$ oc get toolchainstatus -n toolchain-host-operator
+
+NAME               MURS   READY   LAST UPDATED
+toolchain-status   0      True    2021-03-24T22:39:36Z
+```
+and ensure the Ready status is `True`
+
+
+### Register a new KubeSaw User
+
+1. Open the Registration Service URL in a browser and sign up for an account.
+
+2. Wait for the message "Your OpenShift Developer Sandbox account is waiting for approval"
+
+![approval in progress](https://user-images.githubusercontent.com/20015929/114627893-01845d00-9c84-11eb-848e-0f85a1b3c01f.png)
+
+#### Manual Approval
+
+Manual approval means each UserSignup must be approved by editing the UserSignup resource for a particular user.
+
+1. Run the following command to get the name of the UserSignup resource: `oc get usersignup`
+    The name should be a UUID eg. 66e54c45-9868-4a25-81ca-d56b600c8491
+2. Approve the UserSignup
+```
+oc patch usersignup -p '{"spec":{"states":["approved"]}}' --type=merge <usersignup name> -n <host operator namespace>
+```
+
+#### Automatic Approval
+
+By enabling automatic approval in the KubeSaw configuration, users will be automatically approved and provisioned without admin intervention.
+
+Enable automatic approval by running the following command:
+```shell
+oc patch ToolchainConfig -p '{"spec":{"host":{"automaticApproval":{"enabled":true}}}}' --type=merge config -n <host operator namespace>
+```
+
+#### Using KubeSaw instance
+
+After approval, the registration service will display a link to start using the KubeSaw instance. The link will go to the user's Dev Console, but first, a login page will appear with two options.:
+
+1. _kube:admin_
+2. The authentication method configured in the [Authentication step](#authentication)
+![Log in](https://user-images.githubusercontent.com/20015929/114628295-a141eb00-9c84-11eb-8be3-45f013e19378.png)
+Select option 2 and log in using the same account used from the [Register a KubeSaw User section](#register-a-new-kubesaw-user) step.
+
+After logging in, the user will have access to only the namespaces created for them.
+
+### Cleanup
+#### Remove Only Users and Their Namespaces
```

Comment:
Maybe mention here that the make targets are available in the e2e repo?

Reply:
good point, done in https://github.com/kubesaw/kubesaw.github.io/pull/10/commits/f7fc24c415cfc0f65cf0a928707b8fc569578319